composer.json

```diff
@@ -27,6 +27,7 @@
 		"laravel/passport": "^11.0",
 		"laravel/tinker": "^2.0",
 		"laravel/ui": "^4.2",
+		"lcobucci/clock": "^3.2",
```

Comment:
Seems like 3.1 added PHP 8.2 support and 3.2 added PHP 8.3 supporting

With basically nothing else happening between those releases otherwise